rpm/Makefile

```diff
@@ -1,9 +1,8 @@
 ARCH=$(shell uname -m)
 ALPINE_IMG:=$(shell $(CURDIR)/../detect_alpine_image)
-KERNEL:=$(shell docker version --format '{{.Server.Os}}')
+KERNEL:=linux
 ENGINE_DIR:=$(CURDIR)/../../engine
 COMMON_DIR:=$(CURDIR)/../common
```

Comment:
that's correct. We are only using it as a `volume` in order to access the `telemetry script`.